### PR TITLE
Installation script

### PR DESCRIPTION
**Pull request by Vladimir-Z**
Please consider incorporating bash-script. It simplifies installation, a little bit)

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+git clone https://github.com/gmarik/Vundle.vim.git ~/vim-tswr/.vim/bundle/Vundle.vim
+ln -Fs ~/vim-tswr/.vim ~
+ln -fs ~/vim-tswr/.vimrc ~
+vim +PluginInstall +qall
```
Comment (tswr) [inline setup.sh]: No new line at end of file.

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+git clone https://github.com/gmarik/Vundle.vim.git ~/vim-tswr/.vim/bundle/Vundle.vim
```
Comment (tswr) [inline setup.sh]: #!/bin/sh

File: README.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 # Installation
 
     git clone https://github.com/tswr/.vimrc.git ~/vim-tswr/
+    bash ~/vim-tswr/setup.sh
```
Comment (tswr) [inline README.md]: I suggest 

`curl -L https://github.com/tswr/.vimrc/blob/master/setup.sh | sh`

File: README.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
...
+
+The file setup.sh contains following instructions:
```
Comment (tswr) [inline README.md]: These comments are unnecessary, it's github, anyone can look inside of setup.sh :)

File: setup.sh (hunk 1 above)
Comment (tswr) [reply setup.sh]: And maybe installation should be done by install.sh :)

Comment (Vladimir-Z) [reply setup.sh]: I very respect your opinion therefore I'll rename

File: README.md (hunk 4 above)
Comment (Vladimir-Z) [reply README.md]: Hmm.., you are right)

File: README.md (hunk 3 above)
Comment (Vladimir-Z) [reply README.md]: I think It will not work because   https://github.com/tswr/.vimrc/blob/master/setup.sh returns html-file (at least for me)

**Comment by Vladimir-Z** [comment]
url for checking
curl -L https://raw.githubusercontent.com/Vladimir-Z/.vimrc/installation_script/install.sh

**Pull request merged**